airbyte-integrations/connectors/source-convertkit/source_convertkit/manifest.yaml

```diff
@@ -1,86 +1,319 @@
-version: "0.29.0"
+version: 0.79.1
```

Comment:
This still bugs me a little bit. Why is it using 0.79? Current cloud Builder would set 1.3.1 ;)